packages/code-display/src/test-data/specs.ts

```diff
@@ -0,0 +1,48 @@
+/*
+
+ MIT License
+
+ Copyright (c) 2021 Looker Data Sciences, Inc.
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+
+ */
+import { ApiModel, SpecList } from '@looker/sdk-codegen'
+
+export const specs: SpecList = {
+  '3.1': {
+    key: '3.1',
+    isDefault: false,
+    status: 'current',
+    specURL: 'https://self-signed.looker.com:19999/api/3.1/swagger.json',
+    specContent: require('../../../../spec/Looker.3.1.oas.json'),
+    version: '3.1',
+  },
+  '4.0': {
+    key: '4.0',
+    isDefault: true,
+    status: 'experimental',
+    specURL: 'https://self-signed.looker.com:19999/api/4.0/swagger.json',
+    specContent: require('../../../../spec/Looker.4.0.oas.json'),
+    version: '4.0',
+  },
```

Comment:
FWIW the contents of these specs change, which could possibly affect tests that check specific values.

Reply:
Removed. This was vestigial of an old test. Is resolved as I import { codeGenerators } from @looker/sdk-codegen now instead of hard coding the spec